packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -9,6 +9,14 @@ import OrderDetailsFooter from 'Components/order-details/order-details-footer.js
 const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
     const { order_store, sendbird_store } = useStores();
 
+    const showMobileOrderFooter = () => {
```

Comment:
In `order-details-wrapper` , `renderPageFooterChildren` prop renders a footer if a function is passed in, regardless if `should_show_order_footer` is falsy or the function does not return anything, replaced the following code to design requirement:
### Before
<img width="355" alt="Screenshot 2022-04-25 at 11 11 20 AM" src="https://user-images.githubusercontent.com/103016120/165020717-ccca1394-18df-4bf4-b03e-0275e13f0731.png" />

### After
<img width="366" alt="Screenshot 2022-04-25 at 11 10 33 AM" src="https://user-images.githubusercontent.com/103016120/165020752-0ae9cb45-055e-4fd5-8963-6e58a1f5b2d2.png">

### Design

<img width="355" alt="Screenshot 2022-04-25 at 11 11 20 AM" src="https://user-images.githubusercontent.com/103016120/165020856-629df29d-1ef7-4924-8d16-502c63a13495.png">